pilot/pkg/proxy/envoy/v2/xds.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 Istio Authors
```

Comment:
Can you rename this to rds for consistency ( eds goes to eds.go, etc)

Reply:
This can be used for EDS, or LDS, more realistically.